doc/spec_TransferElement.dox

```diff
@@ -158,8 +158,8 @@ This documnent is currently still **INCOMPLETE**!
     - \anchor transferElement_B_8_6_4 8.6.4 Calling TransferElement::interrupt() does not disturb the subsequent operation of the TransferElement. [\ref testTransferElement_B_8_6_4 "T" (base-class part)]
     - \anchor transferElement_B_8_6_5 8.6.5 Calling TransferElement::interrupt() without AccessMode::wait_for_new_data will cause a ChimeraTK::logic_error. [\ref testTransferElement_B_8_6_5 "T"]
 
-- 9. If one transfer element of a device has seen an exception(*), all other transfer elements of the same device must also be aware of this.
-   - \anchor transferElement_B_9_1 9.1 The transfer element which detects the exception reports it to its TransferElement::exceptionBackend by calling DeviceBackend::setException \ref transferElement_comment_B_9_1 "(*)" (cf. \ref transferElement_B_10_1 "10.1")
+- 9. \anchor transferElement_B_9 If one transfer element of a device has seen a ChimeraTK::runtime_error \ref transferElement_comment_B_9a "(*)", all other transfer elements of the same device must also be aware of this. \ref  transferElement_comment_B_9b "(*)"
+   - \anchor transferElement_B_9_1 9.1 The transfer element which detects the ChimeraTK::runtime_error reports it to its TransferElement::_exceptionBackend by calling DeviceBackend::setException() \ref transferElement_comment_B_9_1 "(*)" (cf. \ref transferElement_B_10_1 "10.1")
```

Comment:
I'm kind of missing an explanation what the "_exceptionBackend" is. I assume it is the back-end that handles the exception (so in normal cases the device backend, for LMAP the LMAP backend itself)?

Reply:
As 9.1 say it is the backend to which you report the exception. And yes, it is responsible for executing the actions 9.2, 9.3, 9.4.
This is different from the backend which created the TransferElement. For instance if there are two PCIe-backends in one LNM backend. The TE is directly generated by the PCIe and given out by the LNM.
Now if one of the two PCIes sees an exception, the application only known it talking to the LNM, the only one device it known. So all accessors must be in error state. Hence the exception is reported to the LNM (the exceptionBackend) which delegates the call to all its target backends. Although the Accessor has the PCIe als its internal target on which it calls NumericAddressBackend::read() etc.

Reply:
So mayebe just put he "may be different from the back-end that created the TE, e.g. in aggregating back-ends such as LNM! in a comment?

Reply:
This will be in the changes that Martin requested anyway. I would postpone it to RC8